### ISSUE TITLE
Googlebot smartphone not detected correctly

### ISSUE THREAD

**Issue by samuelpearce** [Wolfcast/BrowserDetection#30]
The following user agent for Googlebot is incorrectly detected as a real browser with no indication that the user is a robot
```
Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
```
This was a user agent I saw in the web logs. More information can be seen [here](https://support.google.com/webmasters/answer/1061943?hl=en)

The [live demo](https://wolfcast.com/open-source/browser-detection/tutorial.php) reports

User agent: | ```Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)```
-- | --
Browser name: | Chrome
Browser version: | 41.0.2272.96
Platform family: | Android
Platform version: | 6.0.1
Platform version name: | Marshmallow
Platform is 64-bit: | No
Is mobile: | Yes
Is robot: | No
IE is in compatibility view: | No
Emulated IE version: | Not applicable
Is Chrome Frame: | No

I would expect the library to report `Is robot: | Yes` as a minimum.
Ideally, the Browser Name should report Googlebot, although it would be a shame to lose the information encoded in the User Agent. I'll leave this up to you :)

Thanks

**Comment by Wolfcast** [comment]
Will fix that ASAP. I will evaluate on how to keep the "emulated" browser by the bot. Maybe like reusing the "Emulated IE version" field that I would rename to "Emulated browser version" which would result in a breaking change but I think it would be a good one.

**Comment by Wolfcast** [comment]
Fixed in 2.9.5 (released just now). Now robot name and version is detected in addition of browser name and version.
